en/organisation/sammansattning/body.md

```diff
@@ -1,179 +1,100 @@
 
-Each board member is the contact person in the board for some of out elected volunteers.
+Each board member is the contact person in the board for some of out elected volunteers a list can be found [here](https://dsekt.se/kontaktfunktionarare).
 
-## The Chairman - David Puustinen
+## The Chairman - Pontus Söderlund
 
 The Chairman is in charge of the D-directorate and the contact with THS and other chapters.
 The Chairman and the Vice Chairman work in close cooperation to create a plan and a vision with the rest of the board.
 
 In addition to this The Chairman is an authorized signatory for the chapter and handles alcohol related questions.
 
-__Contact__
+**Contact**
 
 The Chairman can be reached at [ordf@datasektionen.se](mailto:ordf@datasektionen.se).
 
-__Contact Person to__
+## Vice chairman - Christofer Gärtner
 
-* Head of Reception
-
-* Auditors
-
-## Vice Chairman - Pontus Söderlund
-The Vice Chairmans primary role is to assist the Chairman and make sure
+The Vice chairmans primary role is to assist the Chairman and make sure
 that the every board members workload is reasonable and that they are feeling well.
-In the case that the Chairman can't fulfill his obligations the Vice Chairman will
+In the case that the Chairman can't fulfill his obligations the Vice chairman will
 assume the role as acting Chairman.
 
-The Vice Chairman does many tasks that aren't related to any other board member or function in the chapter.
-
-__Contact__
-
-The Vice Chairman can be reached at [vordf@datasektionen.se](mailto:vordf@datasektionen.se).
-
-__Contact Person to__
-
-* Standard-bearer
-
-* Various project leaders
+The Vice chairman does many tasks that aren't related to any other board member or function in the chapter.
 
-__Contact__
+**Contact**
 
-The Vice Chairman can be reached at [vordf@datasektionen.se](mailto:vordf@datasektionen.se).
+The Vice chairman can be reached at [vordf@datasektionen.se](mailto:vordf@datasektionen.se).
 
-## The Treasurer - Fabian Andréasson
+## The Treasurer - Felix Murnion
 
 The Treasurer is responsible for the most things related to economics.
 This includes declarations, bills, payments, and accounting. The Treasurer
 is an authorized signatory for the chapter.
 
-__Contact__
+**Contact**
 
 The Treasurer can be reached at [kassor@datasektionen.se](mailto:kassor@datasektionen.se).
 
-## Vice Treasurer - Felix Murnion
-The Vice Treasurer assists the Treasurer with most things related to economics.
+## Vice treasurer
 
-__Contact__
+The Vice treasurer assists the Treasurer with most things related to economics.
 
-The Vice Treasurer can be reached at [vkassor@datasektionen.se](mailto:vkassor@datasektionen.se).
+**Contact**
 
-## The Secretary - Herman Karlsson
+The Vice treasurer can be reached at [vkassor@datasektionen.se](mailto:vkassor@datasektionen.se).
 
-The Secretary is responsible for documenting D-directorate and Chapter Meetings
+## The Secretary
```

Comment:
```suggestion
## The Secretary - *(Vacant)*
```
I think it's good to mark that a post is vacant explicitly.

Reply:
I think you're right, but I don't agree with it.